report-hazard/report-hazard.js

```diff
@@ -0,0 +1,96 @@
+const allPages = document.querySelectorAll('div.page');
+allPages[0].style.display = 'block';
+
+function navigateToPage(event) {
+    const pageId = location.hash ? location.hash : '#step1';
+    for (let page of allPages) {
+        if (pageId === '#' + page.id) {
+            page.style.display = 'block';
+        } else {
+            page.style.display = 'none';
+        }
+    }
+    return;
+}
+navigateToPage();
+
+//init handler for hash navigation
+window.addEventListener('hashchange', navigateToPage);
+
+
+class Report {
```

Comment:
rename to ReportForm

Reply:
commit cbe42e108f55532715fcfd714ca3b48283ff65b2